system/exporter/nfs-exporter/templates/deployment.yaml

```diff
@@ -0,0 +1,57 @@
+{{ if .Values.enabled }}
```

Comment:
`enabled` not mentioned in .Values

Reply:
removed the leftover flag it should be enabled / disabled via requirements